### PR TITLE
Update live must gather image for ODF

### PR DESCRIPTION
**Pull request by petr-balogh**
Related BZ:
https://bugzilla.redhat.com/show_bug.cgi?id=2035180

Fixes: #4544

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Review by OdedViner — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Review by sidhant-agrawal — APPROVED** [review]

File: conf/ocsci/odf-live-must-gather.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+# This config file can be used to enforce live must gather image for ODF
+REPORTING:
+  ocs_must_gather_image: "registry.redhat.io/odf4/ocs-must-gather-rhel8"
+  ocs_must_gather_latest_tag: 'latest'
```
Comment (PrasadDesala) [inline conf/ocsci/odf-live-must-gather.yaml:5]: Should we also change the name of the keys from ocs to odf?

**Review by OdedViner — APPROVED** [review]

File: conf/ocsci/odf-live-must-gather.yaml (hunk 1 above)
Comment (petr-balogh) [reply conf/ocsci/odf-live-must-gather.yaml:5]: nope, this is for usecase if you just want to enforce the live image for must gather, we are still taking it from this key in the code.

The logic for --live-deploy parameter is taken from {odf_ocs}_live_must_gather_image which is in default config.

We do not rename usually values from OCS to ODF when code already using this naming convention.

**Review by dahorak — DISMISSED** [review]

**Review by vavuthu — APPROVED** [review]

**Review by dahorak — DISMISSED** [review]

**Review by nehaberry — DISMISSED** [review]
Changes LGTM. Thanks for the quick fix

File: conf/ocsci/odf-live-must-gather.yaml (hunk 1 above)
Comment (PrasadDesala) [reply conf/ocsci/odf-live-must-gather.yaml:5]: Ack

**Review by PrasadDesala — APPROVED** [review]

**Comment by petr-balogh** [comment]
From some reason I see in verification job that it used:
2022-01-07 16:15:42  15:15:40 - MainThread - ocs_ci.ocs.utils - INFO - Must gather image: quay.io/rhceph-dev/ocs-must-gather:latest-4.9 will be used.

Even it's live deployment job.

Will need to check more why it's happening. Adding WIP to subject to prevent this to be merged now.

**Review by OdedViner — APPROVED** [review]

**Review by jimdevops19 — DISMISSED** [review]

**Review by clacroix12 — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**